bambi/families/univariate.py

```diff
@@ -46,27 +66,22 @@ def transform_backend_kwargs(kwargs):
         return kwargs
 
 
-class Binomial(UnivariateFamily):
-    SUPPORTED_LINKS = {"p": ["identity", "logit", "probit", "cloglog"]}
-
-    def posterior_predictive(self, model, posterior, **kwargs):
-        data = kwargs["data"]
-
-        if data is None:
-            trials = model.response_component.response_term.data[:, 1]
-        else:
-            trials = model.response_component.design.response.evaluate_new_data(data)
-
-        response_name = get_aliased_name(model.response_component.response_term)
-        mean = posterior[response_name + "_mean"]
-        return xr.apply_ufunc(np.random.binomial, trials.squeeze(), mean)
+class BetaBinomial(BinomialBaseFamily):
+    # We use the 'mu' and 'kappa' parametrization of the Beta
```

Comment:
Might as well make this a proper class docstring. It would be a good addition for everything else in here as well